src/mdx/_H.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
nit: could we rename this to HeaderTag or Header so it's clear in the code what this renders

Reply:
it renders an h tag :P